packages/babel-helper-module-transforms/src/rewrite-this.js

```diff
@@ -12,10 +13,10 @@ const rewriteThisVisitor = {
     path.replaceWith(path.scope.buildUndefinedNode());
   },
   Function(path) {
-    if (!path.isArrowFunctionExpression()) path.skip();
+    if (!path.isArrowFunctionExpression()) skipAllButComputedKey(path);
```

Comment:
We should call `skipAllButComputedKey` only if `path.isMethod()` (otherwise keep `path.skip()`), because the `skipAllButComputedKey` function expects a class element as parameter.